Algorithms/Dynamic-Programming/Kadane Algorithm.cpp

```diff
@@ -0,0 +1,103 @@
+#include<iostream>
+#include<vector>
+
+using namespace std;
+
+vector<int> maxSubArray(const vector<int> &A)
+{
+    int maxx=INT_MIN,sum=0,i,l=0,r=0,dif=INT_MAX,low_index;
```

Comment:
Try to follow the [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) and use appropriate spacing, also don't declare all the variables in a single line, try to group declarations of similar kind of variables in one line. 
Try to use appropriate names for variables.